### PR TITLE
feat(householdsList): button for add new household

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
feat(householdsList): button for add new household

### CODE REVIEW

File: src/sagas/survey.js [hunk 1]
```diff
@@ -68,28 +59,28 @@ export function* closeSurvey({id}) {
...
         yield put(handleError(err));
     }
 }
 
-export function* findDwelling({id, order}) {
+export function* createHousehold({dwelling}) {
     try {
-        const {survey, dwelling} = yield call(SurveysService.findDwelling, id, order);
-        yield put(receiveDwelling(survey, dwelling));
+        const dwellingUpdated = yield call(SurveysService.addHouseholdToDwelling, dwelling);
```
Comment (navarroaxel) [inline src/sagas/survey.js]: renamed `updatedDwelling`

File: src/services/survey.js [hunk 2]
```diff
@@ -85,21 +85,18 @@ export default class SurveysService {
 
     static async findDwelling(id, order) {
```
Comment (navarroaxel) [inline src/services/survey.js]: `dwellingOrder`

File: src/services/survey.js [hunk 3]
```diff
@@ -85,21 +85,18 @@ export default class SurveysService {
...
         const survey = new Survey(await SurveysService.findById(id));
-        return {
-            survey,
-            dwelling: find(survey.dwellings, dwelling => dwelling.order === toNumber(order))
-        };
+        return find(survey.dwellings, dwelling => dwelling.order === toNumber(order));
```
Comment (navarroaxel) [inline src/services/survey.js]: `const order = toNumber(dwellingOrder);`

File: src/actions/survey.js [hunk 4]
```diff
@@ -86,18 +78,17 @@ export const requestDwelling = (id, order) => ({
...
     type: DWELLING_FETCH_SUCCEEDED,
-    survey,
     dwelling
 });
 
 export const UPDATE_SURVEY_REQUESTED = 'UPDATE_SURVEY_REQUESTED';
 export const UPDATE_SURVEY_SUCCEEDED = 'UPDATE_SURVEY_SUCCEEDED';
 
-export const requestUpdateSurvey = (survey, dwelling) => ({
+export const requestUpdateSurvey = (id, dwelling) => ({
```
Comment (navarroaxel) [inline src/actions/survey.js]: rename `requestUpdateDwelling`

File: src/actions/survey.js [hunk 5]
```diff
@@ -86,18 +78,17 @@ export const requestDwelling = (id, order) => ({
...
     type: UPDATE_SURVEY_REQUESTED,
```
Comment (navarroaxel) [inline src/actions/survey.js]: `SURVEY_UPDATE_REQUESTED`

File: src/components/DwellingResponse/index.js [hunk 6]
```diff
@@ -37,6 +39,7 @@ class DwellingResponse extends Component {
     componentDidMount() {
         const {id, dwelling} = this.props.match.params;
         this.props.requestDwelling(id, dwelling);
+        this.props.requestAddress(id);
     }
 
     componentWillReceiveProps(nextProps) {
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]:
```javascript
if (this.props.saving && !nextProps.saving) {
    this.props.onSubmit(nextProps.survey);
}
```

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**